src/Game/zScript.h

```diff
@@ -7,13 +7,17 @@
 
 struct xScriptAsset : xDynAsset
```

Comment:
The dwarf data says `xScriptAsset` extends `xBaseAsset`: https://github.com/pslehisl/bfbbdecomp/blob/master/dwarf/SB/Game/zScript.cpp#L1106

Why'd you extend xDynAsset?